package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
accidental addition? :sweat_smile:

Reply:
Looks like it.  I was trying to find the name of the dependency that handles the discord components and my cat (yes I'm blaming my new kitten :) ) ran across the keyboard.

Reply:
> yes I'm blaming my new kitten

Fair enough :p (also yay!)